src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs

```diff
@@ -449,6 +456,13 @@ private static Job CreateAotJob(Job baseJob, CommandLineOptions options, Runtime
             return baseJob.WithRuntime(runtime).WithToolchain(builder.ToToolchain());
         }
 
+        private static Job MakeMonoJob(Job baseJob, CommandLineOptions options, MonoRuntime runtime)
+        {
+            return baseJob
+                .WithRuntime(runtime)
+                .WithToolchain(MonoToolchain.From(new NetCoreAppSettings(runtime.MsBuildMoniker, null, runtime.Name, options.CliPath?.FullName, options.RestorePath?.FullName)));

```

Comment:
nit: it would be nice to use named parameters so it's obvious that all strings are passed as right arguments (when they are multiple arguments of the same type it's easy to provide them in wrong order).

You could refactor following logic to a separate method: https://github.com/dotnet/BenchmarkDotNet/blob/c02c3d82bbefa6236660a7b7405f38e739679daf/src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs#L534-L539

Reply:
Done https://github.com/dotnet/BenchmarkDotNet/pull/2142/commits/14a256ef608bbbb228d728747fa50ffa0deb64da